tests/mir-opt/dataflow-const-prop/slice_len.main.DataflowConstProp.32bit.panic-abort.diff

```diff
@@ -30,8 +30,8 @@
           StorageLive(_3);
           StorageLive(_4);
           _14 = const main::promoted[0];
-          _4 = _14;
-          _3 = _4;
```

Comment:
I think you need to update the pass name in `tests/mir-opt/dataflow-const-prop/slice_len.rs`

Reply:
Ah, I also changed all name from `InstSimplify-before-inline` to `InstSimplify-after-simplifycfg`, which can reduce some changes.